pkgs/servers/bililiverecorder/default.nix

```diff
@@ -2,15 +2,15 @@
 , stdenv
 , fetchzip
 , makeWrapper
-, dotnetCorePackages
+, dotnetPackages_6
 }:
 
 let
   pname = "bililiverecorder";
 
-  dotnet = with dotnetCorePackages; combinePackages [
-    runtime_6_0
-    aspnetcore_6_0
+  dotnet = with dotnetPackages_6; combinePackages [
+    runtime
+    aspnetcore
```

Comment:
I know this is how the package was initially conceived, but aspnetcore already implies runtime, it's just runtime with the extra things needed for ASP.NET Core, no?

Reply:
Yeah, that's my understanding.